downstream/modules/test.adoc

```diff
@@ -0,0 +1,39 @@
+// very much a test module
+// Module included in the following assemblies:
+//
+// this module is not used in an assembly. assembly_adding_aws_sources.adoc
+
+// Base the file name and the ID on the module title. For example:
+// * file name: test.adoc
+// * ID: [id="test"]
+// * Title: = Test Module
+
```

Comment:
Looks good!

Reply:
Got it. thanks.

Reply:
I double checked the UI. It is not implemented and will not be documented.